Library/CouponRedemption.cs

```diff
@@ -9,6 +9,7 @@ namespace Recurly
     public class CouponRedemption : RecurlyEntity
     {
 
+        public string UUID { get; private set; }
```

Comment:
@shulmang Looking at this again with this lib fresh on my mind, what would you think about me changing this to `Uuid` to be consistent with the other instance variables?